lib/open_exchange_rates/configuration.rb

```diff
@@ -1,5 +1,25 @@
 module OpenExchangeRates
   class Configuration
     attr_accessor :app_id
+
+    def cache
+      @cache ||= CacheConfiguration.new
+    end
+
+    def cache=(c)
+      @cache = c
+    end
   end
-end
\ No newline at end of file
+
+  class CacheConfiguration
+    attr_accessor :type, :client
+
+    def type
+      @type || 'null'
+    end
+
+    def to_hash
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.